project/Membership.scala

```diff
@@ -32,6 +32,7 @@ trait Membership {
     scalaVersion := "2.11.5",
     resolvers ++= Seq(
       "Guardian Github Releases" at "https://guardian.github.io/maven/repo-releases",
+      "Guardian Github Snapshots" at "http://guardian.github.com/maven/repo-snapshots",
```

Comment:
This extra resolver can probably be removed now in https://github.com/guardian/membership-frontend/pull/1757/files